README.md

```diff
@@ -0,0 +1,16 @@
+# EP
+
+Editor on patterns.
+
+# Building from source
+
+1. Get and build (vcpkg)[https://github.com/microsoft/vcpkg]
+1. The rest is done by `cmake` automatically:
+`cmake -S . -B ./build/ && cmake --build ./build`
```

Comment:
Make readable please

Reply:
Please read from left to right

Reply:
'cmake' occurs 3 times, consider refactor to conform DRY principle

Reply:
Added to backlog yes